integration_tests/e2e/selectDomesticStatus.cy.ts

```diff
@@ -0,0 +1,64 @@
+import { components } from '../../server/@types/contactsApi'
+import TestData from '../../server/routes/testutils/testData'
+import ListContactsPage from '../pages/listContacts'
+import ManageContactDetailsPage from '../pages/manageContactDetails'
+import Page from '../pages/page'
+import SearchPrisonerPage from '../pages/searchPrisoner'
+import SelectDomesticStatusPage from '../pages/selectDomesticStatusPage'
+
+export type PatchContactRequest = components['schemas']['PatchContactRequest']
+
+context('Select Domestic Status', () => {
+  const contactId = 22
+  beforeEach(() => {
+    const { prisonerNumber } = TestData.prisoner()
+    cy.task('reset')
+    cy.task('stubSignIn', { roles: ['PRISON'] })
+    cy.signIn()
+    cy.task('stubComponentsMeta')
```

Comment:
Is it necessary to start from the `prisoner-search/start`? Can we create a parent class or something to abstract away and reuse these steps?
Otherwise, I think can we directly start from the page and avoid these steps ?

Reply:
For the phone ones I just started at "Manage contact"

Reply:
beforeEach amended on integration test starting from Manage contact